mkosi/backend.py

```diff
@@ -833,7 +833,7 @@ def run(
     env: Mapping[str, Any] = {},
     **kwargs: Any,
 ) -> CompletedProcess:
-    cmdline = [str(x) for x in cmdline]
+    cmdline = [os.fspath(x) for x in cmdline]
```

Comment:
Is this correct? I thought we supported passing literally anything to run() and it would be stringified

Reply:
Well, we do `Sequence[PathString]` for the command, which is `Sequence[Union[str, Path]]`, so I think this should be correct.